src/platform/graphics/webgl/webgl-graphics-device.js

```diff
@@ -385,6 +385,8 @@ class WebglGraphicsDevice extends GraphicsDevice {
 
         this.gl = gl;
         this.webgl2 = typeof WebGL2RenderingContext !== 'undefined' && gl instanceof WebGL2RenderingContext;
+        this.isWebGl2 = this.webgl2;
+        this.isWebGl1 = !this.webgl2;
```

Comment:
Should `webgl2` be moved to `deprecated.js`?

Reply:
yes, I will do in a separate PR as it will have many changes.